spec.html

```diff
@@ -25240,39 +25261,49 @@ <h1>
         1. Set the running execution context's LexicalEnvironment to _env_.
         1. Let _instanceExtraInitializers_ be a new empty List.
         1. Let _staticExtraInitializers_ be a new empty List.
+        1. Let _metadataObj_ be ~empty~.
+        1. If _hasDecorators_ is *true*, then
+          1. If |ClassHeritage?| is present, let _metadataParent_ be ? Get(_constructorParent_, @@metadata).
+          1. Else, let _metadataParent_ be *null*.
+          1. Set _metadataObj_ to OrdinaryObjectCreate(_metadataParent_).
```

Comment:
`OrdinaryObjectCreate` only accepts an object or null, but `_constructorParent_[@@metadata]` could be any ECMAScript value.

Probably we shuold:
- if `_constructorParent_[@@metadata]` is undefined, fallback to null
- if it's a different primitive, throw a TypeError

Reply:
Or maybe always throw, given that there is the `Function.prototype[Symbol.metadata]` fallback and thus `undefined` must have been explicitly set.